griptape/drivers/observability/open_telemetry_observability_driver.py

```diff
@@ -18,18 +18,28 @@
 
 @define
 class OpenTelemetryObservabilityDriver(BaseObservabilityDriver):
-    service_name: str = field(kw_only=True)
+    service_name: str = field(default="griptape", kw_only=True)
```

Comment:
Small refactoring opportunity: If we have a default here, then we don't really need it in `GriptapeObservabilityDriver` anymore (which also mean that we need to override `trace_provider` there too). 

Though its fine by me if you don't want to do it in this PR

Reply:
I assumed GriptapeCloud used a non-attribute trace_provider on purpose. If we are OK with GriptapeCloud using the OpenTelemetry tracer_provider, then happy to. Just not clear why it used its own trace_provider in the first place

Reply:
It was just added to work around some attrs awkwardness. The args were left out because gtc doesn't really need them, but they don't hurt either